### PR TITLE
Closes #1: Update `admiraltemplate` to `admiraldata` and copy from admiral.test

### LINKED ISSUE #1: Initial setup of test data
### Background Information

The test data in {admiral.test} needs to be present in {admiraldata}. We will likely make modifications to the data, but to begin with the simplest course of action is to just copy everything over.

### Definition of Done

`R/data.R`, whole of `dev` and `data` folders from {admiral.test} copied into {admiraldata}

### PR DESCRIPTION
**Pull request by kaz462**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiraldev/devel/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/devel/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/cran-release/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by kaz462** [comment]
Following updates in this PR:
1. Copied `R/data.R`, `dev` and `data` folders from `admiral.test` to `admiraldata` 
```
git clone https://github.com/pharmaverse/admiral.test.git --branch devel --single-branch ./admiral.test --depth 1
cp -R ./admiral.test/data  ./admiraldata
cp -R ./admiral.test/dev  ./admiraldata
cp ./admiral.test/R/data.R  ./admiraldata/R
rm -rf admiral.test
```
2. Updated the following and removed some template files & `tests` folder
> 6. Update/Remove the following files to use your extension name:
>    * `DESCRIPTION` File - Name, Authors
>    * `admiraltemplate.Rproj`
>    * `testthat.R`
>    * `News.md`
* `.Rbuildignore`
3. `devtools::document()` + resolve workflow issues

**Review by manciniedoardo — CHANGES_REQUESTED** [review]
@kaz462 is it worth **not** copying the data in the `{admiral.test}` data folder and just regenerating it with the files that you copy over? Just to make sure that what you copied over genuinely makes the dataset that you expect?

File: NEWS.md
Comment (kaz462) [reply NEWS.md]: updated

**Comment by kaz462** [comment]
> @kaz462 is it worth **not** copying the data in the `{admiral.test}` data folder and just regenerating it with the files that you copy over? Just to make sure that what you copied over genuinely makes the dataset that you expect?

I got some errors when I try to rerun the programs from `admiral.test/dev`.
@manciniedoardo Should we merge this PR first, and create a separate issue for rerunning datasets (some programs need updates), so that we can work on other issues in parallel?  

e.g., errors from `dev/pp.R`-
```r
> data("pc")
Warning message:
In data("pc") : data set ‘pc’ not found
> 
> # Calculate subjects with all missing;
> blq_usubjid <- pc %>%
+   ungroup() %>%
+   group_by(STUDYID, DOMAIN, USUBJID) %>%
+   summarise(PPORRES = max(PCSTRESN, na.rm = TRUE))
> 
> # Remove from PC, subjects with all blq (placebos);
> remove_usubjid <- blq_usubjid %>% filter(PPORRES == 0)
> 
> pc1 <- anti_join(pc, remove_usubjid, by = c("STUDYID", "DOMAIN", "USUBJID"))
Error in `anti_join()`:
! Join columns in `y` must be present in the data.
✖ Problem with `STUDYID`, `DOMAIN`, and `USUBJID`.
Run `rlang::last_trace()` to see where the error occurred.
```

**Comment by kaz462** [comment]
@pharmaverse/admiraldata If you are assigned to `admiraldata` issues, please wait until this PR is merged

File: NEWS.md
Comment (manciniedoardo) [inline NEWS.md:1]: change to version 0.1.0

File: NEWS.md
Comment (manciniedoardo) [inline NEWS.md:18]: change to version 0.1.0

**Review by manciniedoardo — CHANGES_REQUESTED** [review]

File: NEWS.md
Comment (kaz462) [reply NEWS.md]: Updated

**Review by manciniedoardo — APPROVED** [review]

**Pull request merged**
